modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2139,19 +2140,27 @@ def broadcast_apply(
         )
         if dtypes == "copy":
             dtypes = self._dtypes
-        new_index = self.index
-        new_columns = self.columns
-        # Pass shape caches instead of values in order to not trigger shape
-        # computation.
-        new_row_lengths = self._row_lengths_cache
-        new_column_widths = self._column_widths_cache
-        if not preserve_labels:
-            if axis == 1:
-                new_columns = joined_index
-                new_column_widths = partition_sizes_along_axis
-            else:
-                new_index = joined_index
-                new_row_lengths = partition_sizes_along_axis
+
+        def _pick_axis(getter, sizes_cache):
```

Comment:
Probably it would be better to have more clearer name for `getter` parameter, smth like `get_axis`

Reply:
renamed, thx